mono/mini/mini-runtime.c

```diff
@@ -2077,7 +2077,15 @@ mono_jit_compile_method_with_opt (MonoMethod *method, guint32 opt, gboolean jit_
 
 		mono_class_init (method->klass);
 
-		if ((code = mono_aot_get_method_checked (domain, method, error))) {
+		code = mono_aot_get_method_checked (domain, method, error);
```

Comment:
mono_aot_get_method_checked () contains very similar code, i.e.:
	if (method->is_inflated && !method->wrapper_type && mono_method_is_generic_sharable_full (method, TRUE, FALSE, FALSE)) {

why is this needed ?